src/middlewared/middlewared/alert/source/cores.py

```diff
@@ -16,6 +16,9 @@ class CoreFilesArePresentAlertClass(AlertClass):
 class CoreFilesArePresentAlertSource(ThreadedAlertSource):
     def check_sync(self):
         cores = "/var/db/system/cores"
+        if self.middleware.call_sync("system.is_enterprise"):
```

Comment:
If you specify `products = ("CORE", "SCALE")` in the class, this alert source won't be shown in enterprise product at all (and won't be ran too)

Reply:
Thanks. I wasn't aware of that.